pkgs/applications/kde/kdenlive/default.nix

```diff
@@ -80,15 +81,19 @@ mkDerivation {
     kpurpose
     kdeclarative
     wrapGAppsHook3
+    glaxnimate
   ];
   # Both MLT and FFMpeg paths must be set or Kdenlive will complain that it
```

Comment:
Comment is outdated now.

Reply:
what do you mean? I never wrote this comment I think ^^

Reply:
Yes, but you changed what the patch does, so the comment needs updating as well.